game/src/plugins/skills/mining/src/ore.kt

```diff
@@ -9,7 +9,7 @@ Thanks to Clifton <http://www.rune-server.org/members/clifton/> for helping
 to find some of the expired object IDs.
  */
 
-data class Ore(val id: Int, val objects: Map<Int, Int>, val level: Int, val exp: Double, val respawn: Int)
+data class Ore(val id: Int, val objects: Map<Int, Int>, val level: Int, val exp: Double, val respawn: Int, val chance: Double, val chanceOffset: Boolean)
```

Comment:
Enum please

Reply:
Again I feel like this over complicates things.

Reply:
Think about it from a development experience perspective: now any time someone wants to use an `Ore` their IDE will make all these values available to them via autocomplete.

Reply:
I agree it looks better.